src/models/TestResultsDAO.ts

```diff
@@ -136,6 +137,10 @@ export class TestResultsDAO {
         vehicleSize,
         vehicleConfiguration,
         vehicleAxles: noOfAxles,
+        euVehicleCategory,
+        vehicleClass,
+        vehicleSubclass,
+        vehicleWheels: numberOfWheelsDriven,
```

Comment:
we can add a console.log for the event.
`console.log("queryString for get Test: ", event)`

Reply:
This will help in cloudwatch logs in debugging quickly.

Reply:
fixed